src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -90,7 +90,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.3.1-SNAPSHOT" />
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.0.2" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.81-preview" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="2.0.12353" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="2.0.12382" />
```

Comment:
Is this the only update. No updates to language workers?

Reply:
No updates for language workers